jasmine/spec/inverted-index-spec.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+var mockFiles = [{
+            "title": "Alice of in Wonderland",
+            "text": "Alice falls into a rabbit shit hole and enters a world full of imagination."
+        },
+
+        {
+            "title": "The Lord of the zip Rings: The Fellowship of the Ring.",
+            "text": "An unusual alliance of man, elf, dwarf, oobit wizard and hobbit seek to destroy a powerful ring."
+        }
+];
+
+//write test to read book data
+describe("Read book data", function () {
+    it("should verify that the files are not empty", function () {
+        expect(mockFiles.length > 0).toBeTruthy();
+    });
+
+    it("should check if all properties in the file are strings", function () {
+        mockFiles.forEach(function (document) {
+            expect(typeof document.title === "string").toBeTruthy();
+            expect(typeof document.text === "string").toBeTruthy();
+        });
+    });
+
+    it("should verify that the file content is a JSON array", function () {
+        expect(Array.isArray(mockFiles)).toBeTruthy();
+    });
+});
+
+describe("Check Class properties", function () {
+beforeEach(function() {
+    this.indexInstance = new Index();
+  });
+
```

Comment:
Nitpick: `beforeEach` has different indentation level from the `it` below. To imporve code quality, strive to have consistent indentation to make the code readable.